server/data/ajaydee.txt

```diff
@@ -544,7 +544,7 @@ The horse galloped uncontrollably.
 The decision was already made.
 They jetted off to the Bahamas.
 The priest collapsed backwards.
-It was a bill for 120 pounds.
```

Comment:
Out of curiosity, this interpretation is specific.  Do we want to have a separate version where the interpretation is "one hundred and twenty"?

Reply:
hi @kscz, what do you mean by keeping a separate interpretation?

Reply:
The sentence could be read both as:
* "It was a bill for one hundred and twenty pounds"
* "It was a bill for a hundred and twenty pounds"

I'm not sure if, in this file, there are similar example like:
* "There were at least one thousand two hundred ducks at the park"
* "There were at least twelve hundred ducks at the park"

I wasn't sure if we were trying to capture the variety of ways people speak a number, or if we're purely trying to train the phonemes for English.

Reply:
gotcha. yeah, that is the exact reason that raw numbers are bad in these sentences. ideally, we would have every possible way to read this number as a separate sentence in this file. but for now, @caugner's fix is a huge improvement to the ambiguity we had before.